test/fail_compilation/impconv.d

```diff
@@ -0,0 +1,36 @@
+/*
+FIXME: DMD host compilers < 2.073 with faulty optimization
```

Comment:
We should have a wiki page with things that can be enabled on the next host compiler bump.
I started one with this PR: https://wiki.dlang.org/Next_Compiler_Bump